Userland/Applications/TerminalSettings/TerminalSettingsWidget.h

```diff
@@ -50,8 +50,10 @@ class TerminalSettingsViewWidget final : public GUI::SettingsWindow::Tab {
     RefPtr<Gfx::Font> m_font;
     float m_opacity;
     String m_color_scheme;
+    VT::CursorShape m_cursor_shape = VT::CursorShape::None;
```

Comment:
```suggestion
    VT::CursorShape m_cursor_shape { VT::CursorShape::None };
```

Please be consistent with the other default-initialized members. Also, should none be the default?

Reply:
Changed the default to block and changed the assignment to use curly braces!